opm/input/eclipse/Parser/Parser.cpp

```diff
@@ -708,8 +708,11 @@ std::optional<std::filesystem::path> ParserState::getIncludeFilePath( std::strin
         std::string stringStartingAtPathName = path.substr(positionOfPathName+1);
         size_t cutOffPosition = stringStartingAtPathName.find_first_not_of(validPathNameCharacters);
         std::string stringToFind = stringStartingAtPathName.substr(0, cutOffPosition);
-        std::string stringToReplace = this->pathMap.at( stringToFind );
-        replaceAll(path, pathKeywordPrefix + stringToFind, stringToReplace);
+
+        if (this->pathMap.find(stringToFind) != this->pathMap.end()) {
+            std::string stringToReplace = this->pathMap.at(stringToFind);
+            replaceAll(path, pathKeywordPrefix + stringToFind, stringToReplace);
```

Comment:
We should probably omit multiple searches.
```suggestion
        if (auto candidate = this->pathMap.find(stringToFind); candidate != this->pathMap.end()) {
            replaceAll(path, pathKeywordPrefix + stringToFind, *candidate);
```

Would also be cool, if you add a test that triggers this.

Reply:
I have now improved the code based on your suggestion. I have also looked for a test file to trigger this error, but I did not find it.